### ISSUE TITLE
Can't Edit value on Grid after update

### ISSUE THREAD

**Issue by MunirMozer** [dbeaver/dbeaver#6242]
#### System information: 
- Operating system (distribution) and version
Windows 10 pro - 1803
- DBeaver version
6.1.2
- Java version 
(build 1.8.0_211-b12)

#### Connection specification:
- Database name and version
SQLSERVER 2016
- Driver name
SQLJDBCDriver
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 
No

#### Describe the problem you're observing:
On version 6.1.1, after execute a select * from table, i can edit the values on grid. 
today, after update 6.1.2, can't edit the column..
roll back the version, edit works fine.

#### Steps to reproduce, if exist:
Open Script
select * from table
try edit a column varchar on grid below, the column is ready only

in version 6.1.1 works fine.

**Comment by serge-rider** [comment]
Can't reprodue.
Could you write exact server/driver versions? (you can get them in Edit connection->Test connection dialog).

- Does your table has primary key or unique index?
- Are there any errors in error/debug logs? (https://github.com/dbeaver/dbeaver/wiki/Log-files)

**Comment by MunirMozer** [comment]
Sorry Serge,
When I did the upgrade, the situation occurred, so I just ran dbeaver-ce-6.1.1-x86_64-setup in the same installation folder. Now, to test the problem again to take the logs, run dbeaver-ce-6.1.2-x86_64-setup in the same folder, but now I can edit the column that I could not before..

Driver: Microsoft JDBC Driver 7.0 SQL Server 7.0.0.0
And yes, my table has a primary key.
![DBeaver 6 1 2 - PK_CFGLOCAL](https://user-images.githubusercontent.com/22770871/60839618-f8f41e00-a1a3-11e9-9e45-fac09d36be56.jpg)

Maybe the problem occurred by any mistake of my.. Keep the good work!
